File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxLen = -1;
    for(let i=0; i<sentences.length; i++){
        let len = 0;
        for(let j=0; j<sentences[i].length; j++){
            if(sentences[i][j] === " "){
                len += 1;
            }
        }
        // console.log(len);
        if(len > maxLen){
            maxLen = len;
        }
    }
    return maxLen+1;
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max = 0;
        for(int i=0; i<sentences.length; i++){
            int count = 0;
            for(int j=0; j<sentences[i].length(); j++){
                if(sentences[i].charAt(j) == ' '){
                    count += 1;
                }
            }
            if(max < count){
                max = count;
            }
        }
        return (max+1);
    }
}